tests/api/entities/contributions.test.js

```diff
@@ -136,6 +136,14 @@ describe('entities:contributions', () => {
       should(patches.find(isEntityPatch(workB)).user).not.be.ok()
       patches.find(isEntityPatch(workC)).user.should.equal(deanonymizedUser._id)
     })
+
+    it('should get non-anonimized contributions for the requesting user', async () => {
```

Comment:
inspired by `tests/api/entities/history.test.js `

```suggestion
    it('should not anonymize contributions when the author is the requesting user', async () => {
```

Reply:
done in 08f30038d